README.md

```diff
@@ -1,58 +1,95 @@
+Required functionality is being added upon request.
 
+Service APIs implemented:
 - Amazon Elastic Compute Cloud (EC2)
 - Amazon Simple Storage Service (S3)
 - Amazon Simple Queue Service (SQS)
-- Amazon SimpleDB
-- Amazon Mechanical Turk
-- Amazon CloudWatch (MON)
-- Amazon DynamoDB (ddb2) 
+- Amazon DynamoDB & DDB streams (ddb2)
+- Amazon Autoscalling(as)
+- Amazon CloudTrail (CT)
+- Cloud Formation (CFN)
+- ElasticLoadBalancing (ELB)
+- Identity and Access Management (IAM)
+- Kinesis
+- CloudWatch
+- MechanicalTurk
+- Simple DB (SDB)
+- Relational Data Service (RDS)
+- Simple Email Service (SES)
+- Short Token Service (STS)
+- Simple Notification Service (SNS)
+- Web Application Firewall (WAF)
+- and more to come
+
+Majority of API functions have been implemented.
+Not all functions have been thoroughly tested, so exercise care when integrating this library into production code.
+Please send issues and patches.
+
+The libraries can be used two ways:
+- either you can specify configuration parameters in the process dictionary. Useful for simple tasks
+- you can create a configuration object and pass that to each request as the final parameter. Useful for Cross AWS Account access
 
-All API functions have been implemented.  Not all functions have been thoroughly tested, so exercise care when integrating this library into production code.  Please send bug reports and patches.
+## Roadmap ##
 
-The libraries can be used two ways: either you can specify configuration parameters in the process dictionary, or you can create a configuration object and pass that to each request as the final parameter.
+Below is the proposed library roadmap update along with regular features and fixes.
 
-## Getting started ##
-You need to clone the repository and download rebar (if it's not already available in your path).
+- v0.13.9
+ * Existing code
 
-```
-git clone https://github.com/erlcloud/erlcloud.git
-cd erlcloud
-wget http://cloud.github.com/downloads/basho/rebar/rebar && chmod u+x rebar
-```
+- v2.0.X
+ * Going further we would like to merge in [Alert Logic](https://github.com/alertlogic/erlcloud/tree/v1.2.3) fork.
+ This is a major version bump which contains lots of new features and functionality.
+ Unfortunately, it also contains quite a number of low level APIs incompatibilities since the fork diverged for a long while.
+ No APIs have been removed and it's on top or 0.13.8 at the moment.
+ Making it backward compatible does not seem feasible and valuable at the moment.
 
-You can provide your amazon credentials in environmental variables.
+- v2.1.X
+ * fix dialyzer findings and make it mandatory for the library
+ * make full support of Mix/HEX
 
+- v2.2.X
+ * Further deprecation of legacy functionality
+  * Only SigV4 signing and generalised in one module. Keep SigV2 in SBD section only
+  * no more `erlang:error()` use and use of regular tuples as error API.
```

Comment:
This sounds like this should entail another major version increment, as it is a substantial behavior change that will again break people's code. I'd make this 3.0.X (or 2.0.X if we decrement the other instances of 2.*.X).